common/configurable/src/test/java/io/helidon/common/configurable/ScheduledThreadPoolSupplierTest.java

```diff
@@ -38,20 +41,28 @@
     private static ScheduledThreadPoolExecutor configuredInstance;
     private static ScheduledThreadPoolExecutor defaultInstance;
 
+    private static ScheduledThreadPoolSupplier builtSupplier;
+    private static ScheduledThreadPoolSupplier configuredSupplier;
+    private static ScheduledThreadPoolSupplier defaultSupplier;
+
+
     @BeforeAll
     static void initClass() {
-        defaultInstance = ScheduledThreadPoolSupplier.create().getThreadPool();
+        ObserverForTesting.clear();
+        defaultSupplier = ScheduledThreadPoolSupplier.create();
+        defaultInstance = defaultSupplier.getThreadPool();
```

Comment:
This test doesn't seem to be shutting down the executors

Reply:
Fixed.